core/src/main/java/org/apache/calcite/sql/SqlJdbcFunctionCall.java

```diff
@@ -752,6 +752,7 @@ private JdbcToInternalLookupTable() {
               return super.createCall(pos, operands);
             }
           });
+      map.put("COUNTIF", simple(SqlLibraryOperators.COUNTIF));
```

Comment:
We likely could implement calcite support for this function, as opposed to merely using `simple`. Not sure on convention — i.e. if changes like this are accepted upstream for mere parseability.

Reply:
Probably don't need COUNTIF in `SqlJdbcFunctionCall`. This class is just for the `{fn COUNTIF()}` syntax, which few people use.